include/net/net_pkt.h

```diff
@@ -2018,6 +2018,12 @@ struct net_pkt *net_pkt_alloc_debug(s32_t timeout,
 #define net_pkt_alloc(_timeout)					\
```

Comment:
Instead of `This is meant for very particular use case. Basically only one uses that: logging.`,
could we say `This is meant for very particular use case as only logging uses that.`
as the latter sounds more English.

Reply:
sure